wslpy/system.py

```diff
@@ -78,4 +78,67 @@ def shellEnvVar(input):
 
 
 def registry(input, key):
-    raise NotImplementedError
\ No newline at end of file
+
+    #DUMMY CODE BEGINS (i am only using this to illustrate something)
+    cmd = u"reg.exe query \""+input+"\" /v \""+key+u"\" 2>&1"
+    p = subprocess.Popen(cmd, shell=True, stdout=subprocess.PIPE)
+    routput, err = p.communicate()
+    #DUMMY CODE ENDS 
+
+    # Given a valid registry path, retrieves the value of an entry in the registry.
```

Comment:
Are there plans to maintain the function's comments with the same format as in the triple quotes?

Reply:
See #15 on discussion with respect to comment format implementation